fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm

```diff
@@ -25,6 +25,8 @@
 	var/warning_target_bloodvol = BLOOD_VOLUME_MAX_LETHAL
 	///Reference to the target we've fed off of
 	var/datum/weakref/target_ref
+	///Are we feeding with aggresive grab or not?
+	var/silent_feed = TRUE
```

Comment:
Comment says it's whether the feed is aggressive, but aggressive shouldn't be silent